storage/extent_store.go

```diff
@@ -150,10 +152,15 @@ func NewExtentStore(dataDir string, partitionID uint64, storeSize, dpType int, i
 	s.dataPath = dataDir
 	s.partitionType = dpType
 	s.partitionID = partitionID
-	if err = MkdirAll(dataDir); err != nil {
-		return nil, fmt.Errorf("NewExtentStore [%v] err[%v]", dataDir, err)
-	}
+
 	if isCreate {
+		if err = s.backupExtent(); err != nil {
```

Comment:
It is not necessary to rename, as it may lead to difficulties in detecting future errors.let it go.